### PR TITLE
Introduce bootstrap concurrency limit

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

**Comment by whyrusleeping** [comment]
@kubuxu  this LGTM, but maybe add a tiny bit more logging so we can see our progress in setting up all the nodes?

File: main.go [hunk 1]
```diff
@@ -113,13 +116,23 @@ func makeAndStartNode(ds ds.Batching, addr string, relay bool, bucketSize int) (
...
+			limiter <- struct{}{}
+		}
 		err = h.Connect(context.Background(), boostrapper())
 		if err != nil {
 			panic(err)
 		}
 
 		time.Sleep(time.Second)
+
+		d.BootstrapOnce(context.Background(), dht.BootstrapConfig{Queries: 4})
```
Comment (lanzafame) [inline main.go]: Can we move all the `context.Background()`s out and have a single root?

Comment (Kubuxu) [reply main.go]: Can we do it as separate PR? I want to get this merged so we can figure out how many nodes we can run on single box.

Comment (lanzafame) [reply main.go]: Sure

**Pull request merged**